src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ConfirmEmail.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@{
+    ViewData["Title"] = "Confirm email";
+}
+
+<h2>@ViewData["Title"]</h2>
+<div>
+    <p>
+        Thank you for confirming your email.
```

Comment:
why there is missing `Please @Html.ActionLink("Click here to Log in", "Login", "Account", routeValues: null, htmlAttributes: new { id = "loginLink" })` part?